### ISSUE TITLE
prism recycler not updated for citadel

### ISSUE THREAD

**Issue by dukrat** [Slivo-fr/EveHQ#136]
The way recycling and fees work has changed in citadel

**Comment by Slivo-fr** [comment]
Is your fix working ?

**Comment by dukrat** [comment]
Yes, I haven't noticed any issues with it.  It would be ideal to add an override faction standings to the page as well, but I couldn't figure out how to edit the UI.  Also I think we might need a new mode: dropdown for those industrial complex things, not sure on that.

**Comment by Slivo-fr** [comment]
Is it something you are willing to work on ?
I can help with the UI I guess
